devilizer/visualizer/display_items.py

```diff
@@ -0,0 +1,23 @@
+def display_items(in_lst: list, current: int) -> str:
+    """Display a list of values.
+
+    Args:
+        in_lst (list): The list of integers to display.
```

Comment:
Maybe change this typehint as well?
```suggestion
        in_lst (list[int]): The list of integers to display.
```